### PR TITLE
Add missing documentation link

### PR DESCRIPTION
**Pull request by yschimke**
security.md and a few others were not linked

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -57,7 +63,7 @@ nav:
     - 'tls': 4.x/okhttp-tls/okhttp3.tls/index.md
     - 'urlconnection': 4.x/okhttp-urlconnection/okhttp3/index.md
     - 'mockwebserver': 4.x/mockwebserver/okhttp3.mockwebserver/index.md
-  - '3.x API':
+  - '3.12.x API':
```
Comment (yschimke) [inline mkdocs.yml]: Tweaked inline with security.md

File: mkdocs.yml [hunk 2]
```diff
@@ -39,15 +39,17 @@ markdown_extensions:
 
 nav:
   - 'Overview': index.md
+  - 'Stack Overflow ⏏': https://stackoverflow.com/questions/tagged/okhttp?sort=active
```
Comment (yschimke) [inline mkdocs.yml]: Made this more front and center, we need more questions here less in issue tracker.

File: mkdocs.yml [hunk 3]
```diff
@@ -39,15 +39,17 @@ markdown_extensions:
...
   - 'Connections': connections.md
-  - 'Recipes': recipes.md
-  - 'Interceptors': interceptors.md
-  - 'HTTPS': https.md
   - 'Events': events.md
-  - 'Works with OkHttp': works_with_okhttp.md
+  - 'HTTPS': https.md
+  - 'Interceptors': interceptors.md
+  - 'Recipes': recipes.md
+  - 'Security': security.md
```
Comment (yschimke) [inline mkdocs.yml]: Added security.md

File: mkdocs.yml (hunk 1 above)
Comment (swankjesse) [reply mkdocs.yml]: I think it’s currently got 3.14.x. I’ll fix manually.

**Pull request merged**